pallets/mb-core/src/lib.rs

```diff
@@ -28,19 +28,18 @@ decl_storage! {
 		GenesisAccounts get(fn genesis_accounts): Vec<T::AccountId>;
 	}
     add_extra_genesis {
-		config(treasury): BalanceOf<T>;
+		config(fntreasury): BalanceOf<T>;
 		config(genesis_accounts): Vec<T::AccountId>;
         build(|config: &GenesisConfig<T>| {
-			<Treasury<T>>::put(config.treasury);
-			// TODO Telmo:  fix
-			// let _ = <GenesisAccounts<T>>::append(config.genesis_accounts.clone());
+			<Treasury<T>>::put(config.fntreasury);
+			let _ = <GenesisAccounts<T>>::put(config.genesis_accounts.clone());
         });
     }
 }
 
 decl_event!(
-	pub enum Event<T>
-	where
+	pub enum Event<T> 
```

Comment:
Is it the auto formatter that adds those spaces ?

Reply:
No, I probably edit this files without selecting the workspace. Just pushed the formatting #c9df4c6.